tools/build_journal.py

```diff
@@ -26,16 +26,24 @@ def main():
         help='Destination host for the payloads')
     parser.add_argument('--destaddr_as_host', action='store_true',
         help='Use destination address as the Host header')
+    parser.add_argument('--protocol', default=None,
+        help='Specify protocol: http or https (default http)')
+    parser.add_argument('--port', default=None,
+        help='Specify port number (default 80)')
```

Comment:
Is this true (default to 80) or whatever is specified in the yaml files?

Reply:
It is true